cmd/npm_publish.go

```diff
@@ -0,0 +1,18 @@
+package main
+
+import (
+	"github.com/grafana/grafana-build/pipelines"
+	"github.com/urfave/cli/v2"
+)
+
+var PublishNPMCommand = &cli.Command{
+	Name:   "publish",
+	Action: PipelineActionWithPackageInput(pipelines.PublishNPM),
+	Usage:  "Using a grafana.tar.gz as input (ideally one built using the 'package' command), take the npm artifacts and publish them on NPM.",
```

Comment:
Is this description accurate for this command?

Reply:
Yup! That's accurate, it takes a package built with the package command and publishes the npm-artifacts inside it.

Reply:
This is to keep the .tar.gz built by the package command as the source of truth, instead of allowing any NPM package to be published by the command.